src/py21cmfast/src/ps.c

```diff
@@ -21,8 +21,12 @@ static gsl_spline *erfc_spline;
 
 #define NSFR_high 200
 #define NSFR_low 250
-#define NGL_SFR 100
-
+//#define NSFR_high 50 //200
+//#define NSFR_low 50 //250
+#define NGL_SFR 100 // 100
+#define NMTURN 50//100
+#define LOG10_MTURN_MAX (double) (10)
+#define LOG10_MTURN_MIN (double) (5.-9e-8)
```

Comment:
Why can't this just be 5.?

Reply:
Isn't LOG10_MTURN_MAX = 10 super large? It should never be much larger than Matom~7-8.

Reply:
I've added -9e-8 for some round error. In most cases, critical mass for photodissociation is smaller than 1e8.5, but for UV ionizing,it can reach ~1e9.6.